src/util/global-config.js

```diff
@@ -0,0 +1,120 @@
+const debug = require('./debug').here(__filename);
+const { resolve } = require('path');
+const { homedir } = require('os');
+const { createHash } = require('crypto');
+const flatfile = require('flat-file-db');
+
+class GlobalConfig {
+    static getDbFilePath() {
+        return resolve(homedir(), './.config/pwa-buildpack.db');
+    }
+    static async db() {
+        if (!this._dbPromise) {
```

Comment:
`this` in a static method is a reference to the `class` itself. If we're storing state that is shared between instances on the class itself, this feels like a singleton. Is there ever a scenario where > 1 `GlobalConfig` instance should be alive?

Reply:
Nah it's fine - just wanted to understand.